lib/src/runner/configuration/load.dart

```diff
@@ -81,10 +79,30 @@ class _ConfigurationLoader {
 
   /// Loads the configuration in [_document].
   Configuration load() => _loadGlobalTestConfig()
+      .merge(_loadIncludeConfig())
```

Comment:
This should be the first in line:

```dart
_loadIncludeConfig()
      .merge(_loadGlobalTestConfig())
      .merge(_loadLocalTestConfig())
      .merge(_loadGlobalRunnerConfig())
      .merge(_loadLocalRunnerConfig());
```

`config1.merge(config2)` gives the fields in `config2` precedence over those in `config1`. We want the local config to be able to override any of the fields in the included one, which means we need to load the included config first and then merge local fields into it.

Reply:
Makes sense. Done!